tools/docker/builder/Dockerfile

```diff
@@ -1,21 +1,23 @@
 FROM gcc:11.2.0@sha256:0eaeaee298c154ecd383cabe5301a5df9fde0cd3a729124caf574500b936389b
 
 ARG RUSTUP_INIT_VERSION=1.25.2
-ARG RUST_VERSION=1.68.0
+ARG RUST_VERSION=1.69.0
```

Comment:
Can you please leave this out if possible, or else mention it more prominently in the PR title if we need to include it? This is kind of off-topic for the concern being tackled here.

From the project-level perspective, ideally we should be using a [Rustup toolchain file](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file) in the repo root as a source of truth, and flowing to other areas from that. I believe this Dockerfile would already properly pick that up with minor changes.

Reply:
> This is kind of off-topic for the concern being tackled here.

Removed